codegen/testserver/resolver.go

```diff
@@ -306,17 +307,38 @@ func (r *userResolver) Friends(ctx context.Context, obj *User) ([]*User, error)
 	panic("not implemented")
 }
 
+// BackedByInterface returns BackedByInterfaceResolver implementation.
 func (r *Resolver) BackedByInterface() BackedByInterfaceResolver { return &backedByInterfaceResolver{r} }
-func (r *Resolver) Errors() ErrorsResolver                       { return &errorsResolver{r} }
-func (r *Resolver) ForcedResolver() ForcedResolverResolver       { return &forcedResolverResolver{r} }
-func (r *Resolver) ModelMethods() ModelMethodsResolver           { return &modelMethodsResolver{r} }
+
+// Errors returns ErrorsResolver implementation.
+func (r *Resolver) Errors() ErrorsResolver { return &errorsResolver{r} }
+
+// ForcedResolver returns ForcedResolverResolver implementation.
+func (r *Resolver) ForcedResolver() ForcedResolverResolver { return &forcedResolverResolver{r} }
+
+// ModelMethods returns ModelMethodsResolver implementation.
+func (r *Resolver) ModelMethods() ModelMethodsResolver { return &modelMethodsResolver{r} }
+
+// OverlappingFields returns OverlappingFieldsResolver implementation.
 func (r *Resolver) OverlappingFields() OverlappingFieldsResolver { return &overlappingFieldsResolver{r} }
-func (r *Resolver) Panics() PanicsResolver                       { return &panicsResolver{r} }
-func (r *Resolver) Primitive() PrimitiveResolver                 { return &primitiveResolver{r} }
-func (r *Resolver) PrimitiveString() PrimitiveStringResolver     { return &primitiveStringResolver{r} }
-func (r *Resolver) Query() QueryResolver                         { return &queryResolver{r} }
-func (r *Resolver) Subscription() SubscriptionResolver           { return &subscriptionResolver{r} }
-func (r *Resolver) User() UserResolver                           { return &userResolver{r} }
+
+// Panics returns PanicsResolver implementation.
```

Comment:
:nauseated_face: 


If you can build them out of a template then its not giving any context or new information to the reader. This is just pure noise.

can we tell golint to ignore these semi-generated files instead?

Reply:
No, We can't... 😿 I agreed about its noisy comment. but I need suppress golint error messages...
golint can't configure that same as gofmt.
golint only have `-min_confidence` options.

Reply:
I highly recommend https://github.com/golangci/golangci-lint.

It will speed up your builds, provide consistent formatting and give you the ability to skip linters consistently on a line/function/file basis based on consistent comments - https://github.com/golangci/golangci-lint#false-positives

Reply:
yes, I know... but Out project can't it immediately.
and, I don't think gqlgen should not limit what tools users should use.
this change is little bit noisy, but useful for golint user I think.